package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "chartjs-plugin-zoom",
   "description": "Simple HTML5 charts using the canvas element.",
-  "version": "0.6.6",
+  "version": "0.7.0",
   "author": "Evert Timberg <evert.timberg@gmail.com>",
```

Comment:
I would remove `author` as we did in the core repo and a few plugins

Reply:
done